vision/module/ModuleKMeans.py

```diff
@@ -0,0 +1,126 @@
+"""
+This class is designed to take an image and binarize it based on what colors
+we need to see.
+"""
+
+import numpy as np
+import cv2
+
+class ModuleKMeans:
+
+    """
+    Applies the kmeans algorithm to an image and displays a remapped
+    version of the image showing either the remapped colors or a black
+    and white image showing only specified color channels
+    Paramters
+    ---------
+    imgPath: string
+        Reads in the image with this name to be used
+
+    channel_weights: float array
+        Decides the weight of each channel in the image
+    """
+
+    def __init__(self, imgPath, channel_weights=[1., 1., 1.]):
+        self.img = cv2.imread(imgPath)
```

Comment:
The init should initialize everything that will be reused in the function. The apply KMeans function should take in an image and process it. This way we create one object and can reuse it for every image.